src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,234 @@
+
+The example below shows the rendering of three discretionary ligatures using the Adobe SourceHanSansJP font:
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="legacy_dlig_jp.png" alt="middle character combination not forming ligature">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 4a, Legacy layout</small><br>
+      MHz correctly forms a ligature<br>
+      but マンション and 10点 does not
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="ng_dlig_jp.png" alt="proper ligatures shown">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 4b, LayoutNG</small><br>
+      All three groups form ligatures as expected
+    </figcaption>
+  </figure>
+</div>
+
+#### Size to Content
+For elements that size-to-content (such as inline blocks) the current layout engine computes the size of the block first and then performs layout on the content. In some cases, such as when a font kerns aggressively, this may result in a mismatch between the size of the content and the block. In LayoutNG this failure mode has been eliminated as the block is sized based on the actual content.
+
+The example below shows a yellow block sized to content. It uses the Lato font which uses kerning to adjust the spacing between T and -. The bounds of the yellow box should match the bounds of the text.
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="kern_legacy.png" alt="trailing whitespace shown at the end of the text container">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 5a, Legacy Layout</small><br>
+      Note the trailing whitespace after the last T
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="kern-ng.png" alt="text bounds have no extra space">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 5b, LayoutNG</small><br>
+      Note how the left and right edges of the box match the bounds of the text
+    </figcaption>
+  </figure>
+</div>
+
+#### Line Wrapping
+Similar to the problem described above, if the content of a size-to-content block is larger (wider) than the block, this can cause the content to wrap unnecessarily. This is quite rare but sometimes happens for mixed directionality content.
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="legacy_ar_wrap.png" alt="premature line break shown causing extra space">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 6a, Legacy layout</small><br>
```

Comment:
```suggestion
      <small>Fig 6a, Legacy layout engine</small><br>
```

Reply:
if we change this here we should change it everywhere no?

Reply:
looks like you did and i was too quick to comment. thanks!